app/uk/gov/hmrc/alcoholdutyreturns/connector/ReturnsConnector.scala

```diff
@@ -62,4 +62,31 @@ class ReturnsConnector @Inject() (
           Left(ErrorResponse.UnexpectedResponse)
       }
   )
+
+  def submitReturn(returnToSubmit: ReturnCreate, appaId: String)(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, ErrorResponse, ReturnCreatedDetails] = EitherT(
+    httpClient
+      .POST[ReturnCreate, Either[UpstreamErrorResponse, HttpResponse]](
+        url = config.submitReturnUrl,
+        body = returnToSubmit,
+        headers = headers.submitReturnHeaders(appaId)
+      )
+      .map {
+        case Right(response)                                                =>
+          Try(response.json.as[ReturnCreatedSuccess]).toOption
```

Comment:
As mentioned in team channel, this loses the validation error context. You can use something like:

      Try {
              response.json
                .as[FinancialTransactionDocument]
            } match {
              case Success(doc)       =>
                logger.info(s"Retrieved financial transaction document for appaId $alcoholDutyReference")
                Some(doc)
              case Failure(exception) =>
                logger.warn(
                  s"Parsing failed for financial transaction document with appaId $alcoholDutyReference",
                  exception
                )
                None
            }